frontend/src/modules/entitieslist/components/EntitiesList.test.js

```diff
@@ -11,7 +11,10 @@ import * as batchactions from 'modules/batchactions';
 import EntitiesList, { EntitiesListBase } from './EntitiesList';
 
 // Entities shared between tests
-const ENTITIES = [{ pk: 1 }, { pk: 2 }];
+const ENTITIES = [
+    { pk: 1, translation: [{ string: '', errors: [], warnings: [] }] },
```

Comment:
Because we are now deep rendering (i.e. `.mount()`), the passed in state needs to have more members so that children components don't crash and can render.